project/main/templates/main/index.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'main/base.html' %}
+
+{% block title %}SIG{% endblock %}
```

Comment:
Shouldn't this be:
```html
<title>
   SIG
</title>
```

Reply:
The default title, if any will always be included from base.html. This structure helps in extending the same. For ex:
'ACM SIG- ' is common for all pages; then for the Competetive SIG page we just need to input 'Competetive SIG' between those tags, the page will finally be rendered with title: 
ACM SIG-Competetive SIG